modules/troubleshooting-network-observability-loki-large-query-timeout.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-troubleshooting-large-query-timeout_{context}"]
+= Running a large query results in a Loki timeout error
+When running large queries for a long time, a Loki timeout error occurs. There is no complete corrective for this issue, but there are several ways to mitigate it:
+
+Adapt your query to add an indexed filter::
++
+With Loki queries, you can query on both indexed and non-indexed fields, or labels. Queries that contain filters on labels perform better. For example, if you query for a particular Pod that is not indexed, you can add its Namespace to the query. The list of indexed fields can be found in the "Network flows format reference", in the `Loki label` column.
```

Comment:
This could be confusing when saying "if you query for a particular Pod that is not indexed", making think that some pod names might be indexed and others not - that's not the case, none of the pods are indexed.

```suggestion
With Loki queries, you can query on both indexed and non-indexed fields, or labels. Queries that contain filters on labels perform better. For example, if you query for a particular Pod, which is not an indexed field, you can add its Namespace to the query. The list of indexed fields can be found in the "Network flows format reference", in the `Loki label` column.
```